changelog/Microsoft.DirectoryServices.json

```
@@ -2,14 +2,24 @@
   "changelog": [
     {
       "ChangeList": [
-        {
-          "Id": "31a0a04f-7706-4c72-94b7-752932a6592e",
-          "ApiChange": "Resource",
-          "ChangedApiName": "identityProviders",
-          "ChangeType": "Addition",
-          "Description": "Added the operation [availableProviderTypes](https://docs.microsoft.com/graph/api/resources/identityprovider-list-availableprovidertypes.md?view=graph-rest-1.0&preserve-view=true) to list the available identity providers available for use in an Azure Active Directory tenant.",
-          "Target": "availableProviderTypes"
-        },
+      { 
+        "Id": "31a0a04f-7706-4c72-94b7-752932a6592e",
+        "ApiChange": "Resource",
+        "ChangedApiName": "identityProviders",
+        "ChangeType": "Addition",
+        "Description": "Added the operation [availableProviderTypes](https://docs.microsoft.com/graph/api/resources/identityprovider-list-availableprovidertypes.md?view=graph-rest-1.0&preserve-view=true) to list the available identity providers available for use in an Azure Active Directory tenant.",
+        "Target": "availableProviderTypes"
+      }
+    ],
+      "Id": "66d29d1a-7dc5-4ca7-84f2-a33b95ead810",
```

Minor change. Same GUID for the changelog object.
```suggestion
      "Id": "31a0a04f-7706-4c72-94b7-752932a6592e",
```